src/export/Package.ts

```diff
@@ -47,9 +67,25 @@ export class Package implements Fishable {
           break;
         case Type.ValueSet:
           def = this.valueSets.find(vs => vs.id === item || vs.name === item || vs.url === item);
+          if (!def) {
+            def = this.instances.find(
+              i =>
+                (i.id === item || i._instanceMeta.name === item || i.url === item) &&
+                i._instanceMeta.usage === 'Definition' &&
+                i.resourceType === 'ValueSet'
+            );
+          }
           break;
         case Type.CodeSystem:
           def = this.codeSystems.find(cs => cs.id === item || cs.name === item || cs.url === item);
+          if (!def) {
+            def = this.instances.find(
+              i =>
+                (i.id === item || i._instanceMeta.name === item || i.url === item) &&
+                i._instanceMeta.usage === 'Definition' &&
+                i.resourceType === 'CodeSystem'
```

Comment:
And under the assumption that _most_ instances are not CodeSystems, it might be more efficient to check `i.resourceType === 'CodeSystem'` first.